### DISCUSSION TITLE
Url should not always have /admin in it

### DISCUSSION THREAD

**Discussion by cloudcompute** [directus/directus#12889] [Feature Requests]
Hi

For both admin and non-admin users, the Url looks like
127.0.0.1:8055/admin/login

At time of login it should be:
127.0.0.1:8055/login

And after login: /login part should not be there.
127.0.0.1:8055/admin/login

If he is a non-admin user, this may confuse him.

### To Reproduce

Type 127.0.0.1:8085 in the Url

and the Url immediately changes to

127.0.0.1:8055/admin/login

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
